### ISSUE TITLE
Shortcut (Alt+Shift+C) shown in toolbar button hint not working in Firefox

### ISSUE THREAD

**Issue by Wolf-at-SO** [asamuzaK/url2clipboard#36]
The shortcut is shown in the hint of the toolbar icon but seems to be already occupied by Firefox itself. I searched the setting for an option to change the shortcut but found noting.

Maybe this flaw is restricted to my computer only, but if the keyboard shortcut doesn't work in all FF 57.0.2 (64 Bit) installations under Windows 8.1, it should at least be removed from the hint.

**Comment by asamuzaK** [comment]
Can not confirm the issue.
Shortcut is working fine on Firefox Nightly 59.0a1, Firefox Developer Edition 58.0b10, Firefox ESR 52.5.2 (on Windows 10)

Maybe conflict with another add-on?

**Comment by Wolf-at-SO** [comment]
Hm, thanks for having a look on it. It's FF's browser history that pops up in a context menu, so I think it's an issue for German users, there it is named *Chronik*. Would it be possible to make the shortcurt configurable?

**Comment by asamuzaK** [comment]
It is possible to implement keyboard shortcut customization.
Actually I am doing that with another add-on withExEditor.

However, customized shortcuts will not work with URLs where content scripts are restricted, such as addons.mozilla.org.
Thinking about the way this add-on is used, I do not think that such implementation is a good idea.

[1303384 - UI for re-assigning command shortcuts](https://bugzilla.mozilla.org/show_bug.cgi?id=1303384 "1303384 - UI for re-assigning command shortcuts")
Once this bug is fixed, users can customize shortcuts.

**Comment by asamuzaK** [comment]
Can I close this?

**Comment by Wolf-at-SO** [comment]
It's not an important issue *for me* but maybe for others, because I find it easier to use the functions from context menu (which, BTW, has its own key on most keyboards). 

Another point: is it a good idea to use the combination Alt+Shift+ at all, isn't it (sort of) reserved for [access keys](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey) for content?

IOW: is a keyboard shortcut really important? If you'd decide to drop it, this issue gets invalid automatically.

**Comment by Wolf-at-SO** [comment]
Oops: now it works also in my browser. So what now?

1) the issue in its current form is invalid now. (WORKSFORME)

2) the [`accesskey` web feature](https://www.w3.org/TR/2016/REC-html51-20161101/editing.html#the-accesskey-attribute) should be considered

I'm not lucky with keyboard shortcuts that work only if no access key is present: how can you trust in a hotkey if you must be prepared for rare (and surprising) impacts within some web app?

Anyway: thanks for your time.

**Comment by asamuzaK** [comment]
Access key attributes always have a risk of conflict with OS shortcuts.
So, it's not only the extensions problem.

In my personal experience, access keys were often assigned to pages for old mobile phones (like 4 for history back, 6 for forward), but I rarely see them used on recent web pages (although I may not be aware of them) .

**Comment by Wolf-at-SO** [comment]
If you are working on MediaWiki-based sites, accesskeys are crucial ([e: edit; p: preview; v: diff; s: save](https://www.mediawiki.org/wiki/Manual:Interface/Access_keys)) at least in my personal experience.

**Comment by asamuzaK** [comment]
Thanks.

**Comment by asamuzaK** [comment]
The accesskey attribute has been changed several times in HTML 4, HTML 5, HTML 5.1, and key combination varies from browser to browser.
In addition, Firefox switched it's key combination several times.

In my opinion, websites and web applications should avoid using the accesskey attribute.
It is not accessible at all, contrary to its name.

I decided that considering the accesskey attribute on the add-on side is meaningless and decided to keep the keyboard shortcut as is.
